docs/docs-content/clusters/cluster-management/platform-settings/pause-platform-upgrades.md

```diff
@@ -8,11 +8,58 @@ tags: ["clusters", "cluster management"]
 ---
 
 Palette supports the **Pause Agent Upgrades** feature to exclude a cluster or a group of clusters from getting
```

Comment:
this isn't correct, it is NOT about upgrading the cluster itself, only the agent that runs in the cluster